monkestation/code/modules/mob/living/emote.dm

```diff
@@ -0,0 +1,31 @@
+/datum/emote/living/thumbs_up
+	key = "thumbsup"
+	key_third_person = "thumbsup"
+	message = "flashes a thumbs up"
+	message_robot = "makes a crude thumbs up with their 'hands'"
+	message_AI = "flashes a quick hologram of a thumbs up"
+	message_ipc = "flashes a thumbs up icon"
+	message_simple = "attempts a thumbs up"
+	message_param = "flashes a thumbs up at %t"
+
+/datum/emote/living/thumbs_down
+	key = "thumbsdown"
+	key_third_person = "thumbsdown"
+	message = "flashes a thumbs down"
+	message_robot = "makes a crude thumbs down with their 'hands'"
+	message_AI = "flashes a quick hologram of a thumbs down"
+	message_ipc = "flashes a thumbs down icon"
+	message_simple = "attempts a thumbs down"
+	message_param = "flashes a thumbs down at %t"
+
+/datum/emote/living/whistles
```

Comment:
This is "whistles" while the get_sound is "whistle"

Reply:
ohh thats why it didnt work